packages/commercetools/api-client/src/helpers/createCommerceToolsLink/index.ts

```diff
@@ -19,7 +20,22 @@ const createCommerceToolsLink = (): ApolloLink => {
     };
   });
 
-  return ApolloLink.from([authLink, httpLink]);
+  const onErrorLink = onError(({ graphQLErrors, networkError }) => {
+    if (graphQLErrors) {
+      graphQLErrors.map(({ message, locations, path }) => {
+          const parsedLocations = locations.map(({ column, line }) => `[column: ${column}, line: ${line}]`);
+
+          console.log(`[GraphQL error]: Message: ${message}, Location: ${parsedLocations.join(', ')}, Path: ${path}`);
+        }
+      );
+    }
+
+    if (networkError) {
+      console.log(`[Network error]: ${networkError}`);
```

Comment:
Same here, `console.error`

```suggestion
      console.error(`[Network error]: ${networkError}`);
```